addons/hr_timesheet/wizard/project_task_create_timesheet.py

```diff
@@ -41,4 +41,5 @@ def save_timesheet(self):
             'timer_pause': False,
             'timesheet_timer_last_stop': fields.datetime.now(),
         })
+        self.task_id.user_timer_id.unlink()
```

Comment:
why unlinking the timer here ?

Reply:
@LucasLefevre , because it's useless to keep the stopped timer. A new timer can be create if needed. and in the super of action_timer_stop it was going to unlink so we unlinked it over here. Thanks